content/millennium/bulk-data/group-export.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Group Export | R4 API
+---
+
+# Group Export
+
+<%= beta_tag %>
+
+* TOC
+{:toc}
+
+## Overview
+
+The Group Export operation is used to obtain FHIR resources for all members of a specified group of patients.
+
+An export process uses the [FHIR Asynchronous Request Pattern](https://hl7.org/fhir/R4/async.html) to kick off the export job for a group of patients. Once the export job is kicked off, the status endpoint may be called to check the progress of the job. Once the job is complete, the status endpoint returns a list of files which may be downloaded using the file retrieval endpoint. After the appropriate ndjson files are downloaded, the delete endpoint should be called to clean up the no longer needed files.
```

Comment:
list of files which may be downloaded using the file retrieval endpoint  --> list of file URLs which may be called to download the content of the file.

Reply:
Updated here: https://github.com/cerner/fhir.cerner.com/pull/899/commits/eff91ed66335ec7340cb5a2ec8a51663c1bd9cbf#diff-c2a4803bbbf3b17e514d41cc6eb7d578c94a283f937cce0400f5437570744d7fR16